### PR TITLE
consolidate `conda_build` in `conda-forge.yml`

### LINKED ISSUE #21: Consolidate duplicate `conda_build` keys in `conda-forge.yml`
It looks like there are duplicate `conda_build` keys in `conda-forge.yml`

https://github.com/conda-forge/pyclesperanto-feedstock/blob/3dc80c8041a186497473b7dc055cccace2d8fef6/conda-forge.yml#L5-L6

https://github.com/conda-forge/pyclesperanto-feedstock/blob/3dc80c8041a186497473b7dc055cccace2d8fef6/conda-forge.yml#L11-L12

Would it be possible to consolidate this content under one `conda_build` key and drop the other?

This would help fix some issues the bot (used for migrations and version updates) is seeing

### PR DESCRIPTION
**Pull request by StRigaud**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

closes #21

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% set name = "pyclesperanto" %}
 {% set version = "0.11.1" %}
-{% set build_number = 0 %}
+{% set build_number = 1 %}
```
Comment (jakirkham) [inline recipe/meta.yaml:3]: This isn't strictly needed as we haven't changed the package. Though no harm in including it

**Review by jakirkham — COMMENTED** [review]
Thanks Stephane! 🙏

Looks good 😀

**Comment by StRigaud** [comment]
Again, thanks for pointing it to me ! 😃

**Comment by jakirkham** [comment]
Happy to help 🙂

Thanks for the prompt fix! 🙏

**Pull request merged**